src/test/scala/chiselTests/FixedPointSpec.scala

```diff
@@ -0,0 +1,46 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import chisel3._
+import chisel3.testers.BasicTester
+import org.scalatest._
+
+//scalastyle:off magic.number
+
```

Comment:
Is this scalastyle comment used? If so I guess there shouldn't be a line between it and the class?